elementary/artifacts/entities/test_results.py

```
@@ -0,0 +1,214 @@
+import sys
+from typing import Any, Dict, List, Optional
```

Flake8 (7.3.0)

[error] 'typing.Optional' imported but unused

(F401)